content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -104,6 +104,50 @@ The cluster autoscaler never evicts Pods that have this annotation explicitly se
 `"false"`; you could set that on an important Pod that you want to keep running.
 If this annotation is not set then the cluster autoscaler follows its Pod-level behavior.
 
+### config.kubernetes.io/local-config
+
+Example: `config.kubernetes.io/local-config: "true"`
+
+Used on: All resources
+
+This annotation is used to mark a resource as a local configuration file that should not be applied to the API server. It is part of the [Kubernetes Resource Model (KRM) Functions Specification](https://github.com/kubernetes-sigs/kustomize/blob/ce3e394a414387ce09679523902e981414b09a1a/cmd/config/docs/api-conventions/functions-spec.md) that is used by tools such as Kustomize and Kpt. For example, Kustomize removes objects with this annotation from its final build output.
```

Comment:
Is this ok to link to, or similar to the KEP in my other PR, do we need to move the full content itself into the docs site (and if so, where?)?

Reply:
For Kustomize, how about either:
- https://kubectl.docs.kubernetes.io/references/kustomize/resource-model/
- https://kubernetes.io/docs/reference/kubectl/kustomize-resource-model/

Lop bits off those URLs to find valid parent pages.

Reply:
We can omit the link for now and file an issue to put it in later.

Reply:
(if you do file an issue, please mention it here)

Reply:
NB KRM is **Kubernetes** Resource Model (not Kustomize)

There are three potential concepts to cover here :
- KRM: [Kubernetes Resource Model](https://github.com/kubernetes/design-proposals-archive/blob/main/architecture/resource-management.md#the-kubernetes-resource-model-krm), a general high-level k8s concept not at all specific to Kustomize
- KRM Functions Specification: an open standard that tools like Kustomize (called "function orchestrators" in this context) can implement as the basis for an extensions ecosystem.
- KRM Function: an instance of a client-side extension that complies with the KRM Functions Specification and can therefore be used to extend compatible (function orchestrator) tools.

Do we want docs for any/all of those? Or just brief definitions perhaps?